plugins/woocommerce-admin/client/products/sections/images-section.tsx

```diff
@@ -161,43 +173,83 @@ export const ImagesSection: React.FC = () => {
 							</CardBody>
 						) : (
 							<CardBody>
-								<MediaUploader
-									onError={ () => null }
-									onSelect={ ( file ) => {
-										if (
-											images.find(
-												( img ) => file.id === img.id
-											) === undefined
-										) {
-											setValue( 'images', [
-												...images,
-												file,
-											] );
-										}
-									} }
-									onUpload={ ( files ) => {
-										if ( files[ 0 ].id ) {
-											setValue( 'images', [
-												...images,
-												...files,
-											] );
-										}
+								<FormFileUpload
+									accept="image/*"
+									multiple={ true }
+									onChange={ ( { currentTarget } ) => {
+										uploadMedia( {
+											filesList:
+												currentTarget.files as FileList,
+											onError: () => null,
+											onFileChange: onFileUpload,
+											maxUploadFileSize: 10000000,
+										} );
 									} }
-									label={
-										<>
-											<img
-												src={ DragAndDrop }
-												alt="Completed"
-												className="woocommerce-product-form__drag-and-drop-image"
+									render={ ( { openFileDialog } ) => (
+										<div
+											className="woocommerce-form-file-upload"
+											onKeyPress={ () => {} }
+											tabIndex={ 0 }
+											role="button"
+											onClick={ (
+												event: React.MouseEvent<
+													HTMLDivElement,
+													MouseEvent
+												>
+											) => {
+												const { target } = event;
+												if (
+													(
+														target as HTMLButtonElement
+													 )?.type !== 'button'
+												) {
+													openFileDialog();
+												}
+											} }
+											onBlur={ () => {} }
+										>
+											<MediaUploader
+												onError={ () => null }
+												onSelect={ ( file ) => {
+													if (
+														images.find(
+															( img ) =>
+																file.id ===
+																img.id
+														) === undefined
+													) {
+														setValue( 'images', [
+															...images,
+															file,
+														] );
+													}
+												} }
+												onUpload={ ( files ) => {
+													if ( files[ 0 ].id ) {
+														setValue( 'images', [
+															...images,
+															...files,
+														] );
+													}
+												} }
+												label={
+													<>
+														<img
+															src={ DragAndDrop }
+															alt="Completed"
```

Comment:
Looks like we missed localizing this one previously:

```suggestion
															alt={ __( 'Completed', 'woocommerce' ) }
```